docs/webpack-dev-server.config.js

```diff
@@ -39,6 +39,8 @@ const config = {
   plugins: [
     // Allows for sync with browser while developing (like BrowserSync)
     new webpack.HotModuleReplacementPlugin(),
+    // Prevent moment from loading all the locales
+    new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/),
```

Comment:
Do we need to mention this in the docs, so that anyone using the Persian lib doesn't get hit by it?